### ISSUE TITLE
Demo4 causes a dump with new CONVERT_COLUMN2INT method

### ISSUE THREAD

**Issue by ivanfemia** [abap2xlsx/abap2xlsx#162]

**Comment by ivanfemia** [comment]
(In [[r:299]]) Ready to test #163
Ready to test #164

**Comment by ivanfemia** [comment]
I don't think the current fix can work out.

1st)  The lv_column CN '1234567890'
Here I guess the operator should be "CO" instead of "CN".  Furthermore it won't work if you have a char-variable with trailing or leading blanks

2nd)  The dump results from method CONVERT_COLUMN2INT raising an exception if column "0" is passed.  Basically you should allow "0" as allowed column if that really is wanted.

The origin of "0" being passed in demo4 is that the 3rd (hidden) sheet does not contain any filled cells.  Because of this the writer somewhere calls method ZCL_EXCEL_WORKSHEET->UPDATE_DIMENSION_RANGE and here the first entry in the content table is being read.  Due to the sheet being empty this results in a subrc = 4 which is not caught which consequently leads to the dump you encounter.

I'd propose to fix the latter method instead of changes to the CONVERT_COLUMN2INT, because right now passing a "+4" will probably result in an incorrect column.

**Comment by ivanfemia** [comment]
Stefan,

your're right, I committed a new version just after few mins :) Let me know if the commit is not correct.

0 is not a valid column as we consider column A as column #1 as for OXML, I have also fixed method ZCL_EXCEL_WORKSHEET->UPDATE_DIMENSION_RANGE in order to avoid the calling of method COLUMN2INT if no cell is provided

Btw value "+4" is not a valid input, it should be "4". Why we should use a sign? Negative value are not valid.

Let me know if the latest commit works.

Thanks 
Ivan
